### DISCUSSION TITLE
Any way to make a color transparent or delete a property color?

### DISCUSSION THREAD

**Discussion by oneeyedreaper** [ryanraposo/codeui#43] [Q&A]
I'm ttrying to put the hex color with alpha 0 but it's not supported. Any suggestions?

**Comment by ryanraposo** [comment]
> I'm ttrying to put the hex color with alpha 0 but it's not supported. Any suggestions? 

@oneeyedreaper Could you give me an example? What does your input look like exactly?

**Reply by oneeyedreaper** [reply]
I clicked on the customize button in front of the color and opted to add a custom value. Added the value in #AARRGGBB format and a popup came up saying "is not a valid hex color".

**Reply by ryanraposo** [reply]
> I clicked on the customize button in front of the color and opted to add a custom value. Added the value in #AARRGGBB format and a popup came up saying "is not a valid hex color".

See #44, there is a major bug causing customizations to fail.

However, using the workaround described in that Issue, I am able to input alpha hex values.

Your issue appears to be separate so I need more info:

- Your current theme
- The exact hex value you are using

Thanks!

**Reply by oneeyedreaper** [reply]
Thanks for mentioning that, I added the workbench.colorCustomizations manually later and it's working now.
I'm currently using the Gruvbox Glass theme with Vibancy and the color I was using was #ffffff00.
